src/Dev/Command/DevCommand.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace SilverStripe\Dev\Command;
+
+use SilverStripe\PolyExecution\PolyCommand;
+use SilverStripe\PolyExecution\PolyOutput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Terminal;
+
+/**
+ * A command that can be run from CLI or via an HTTP request in a dev/* route
+ */
+abstract class DevCommand extends PolyCommand
+{
+    private static array $permissions_for_browser_execution = [
+        'ADMIN',
+        'anyone_with_dev_admin_permissions' => 'ALL_DEV_ADMIN',
```

Comment:
```suggestion
        'ALL_DEV_ADMIN',
```
I found the additional of 'helpful' keys just confusing, I thought they were adding some additional functionality when they're not

Reply:
See https://github.com/silverstripe/silverstripe-framework/pull/11353#discussion_r1772353673